.github/workflows/tests_linux.yml

```diff
@@ -50,7 +50,9 @@ jobs:
 
       - name: Install Latest PennyLane
         if: inputs.pennylane-version == 'latest'
-        run: python -m pip install git+https://github.com/PennyLaneAI/pennylane.git@master
+        run: |
+          python -m pip install --index-url https://test.pypi.org/simple/ pennylane-lightning --pre --force-reinstall --no-deps
```

Comment:
I'm OK with installing the most recent Lightning Qubit package, but is this needed for some reason?

Reply:
To be sure to pull a version of Lightning that has the latest fix, it is better force it. For example, [this workflow](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/actions/runs/5968205845/job/16191528990#step:6:113) is failing since PennyLane pulls Lightning v0.31.